### ISSUE TITLE
Failing command:  brew cask style

### ISSUE THREAD

**Issue by mihaicris** [caskroom/homebrew-cask#45650]
#### General troubleshooting steps

- [x] I have retried my command with `--force` and the issue is still present.
- [x] I have checked the instructions for [reporting bugs](https://github.com/caskroom/homebrew-cask#reporting-bugs) (or [making requests](https://github.com/caskroom/homebrew-cask#requests)) before opening the issue.
  - [x] None of the templates was appropriate for my issue, or I’m not sure.
- [x] I ran `brew update-reset && brew update` and retried my command.
- [x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).

#### Description of issue

Failing command
```
brew cask style
```

#### Output of your command with `--verbose --debug`

```
12:23 $ sudo gem install did_you_mean -v 1.0.0
Password:
Successfully installed did_you_mean-1.0.0
Parsing documentation for did_you_mean-1.0.0
Done installing documentation for did_you_mean after 0 seconds
1 gem installed

12:23 $ brew cask style --verbose --debug
==> Installing or updating 'rubocop-cask' gem
Error: Unable to resolve dependency: user requested 'did_you_mean (= 1.0.0)'
Follow the instructions here:
  https://github.com/caskroom/homebrew-cask#reporting-bugs
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:231:in `search_for'
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:283:in `block in sort_dependencies'
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:277:in `each'
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:277:in `sort_by'
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:277:in `with_index'
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:277:in `sort_dependencies'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/delegates/specification_provider.rb:52:in `block in sort_dependencies'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/delegates/specification_provider.rb:69:in `with_no_such_dependency_error_handling'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/delegates/specification_provider.rb:51:in `sort_dependencies'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/resolution.rb:165:in `initial_state'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/resolution.rb:106:in `start_resolution'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/resolution.rb:64:in `resolve'
/Library/Ruby/Site/2.3.0/rubygems/resolver/molinillo/lib/molinillo/resolver.rb:42:in `resolve'
/Library/Ruby/Site/2.3.0/rubygems/resolver.rb:188:in `resolve'
/Library/Ruby/Site/2.3.0/rubygems/request_set.rb:385:in `resolve'
/Library/Ruby/Site/2.3.0/rubygems/request_set.rb:397:in `resolve_current'
/Library/Ruby/Site/2.3.0/rubygems.rb:243:in `finish_resolve'
/Library/Ruby/Site/2.3.0/rubygems/rdoc.rb:15:in `<top (required)>'
/Library/Ruby/Site/2.3.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/Library/Ruby/Site/2.3.0/rubygems/core_ext/kernel_require.rb:59:in `require'
/Library/Ruby/Site/2.3.0/rubygems/commands/install_command.rb:280:in `load_hooks'
/Library/Ruby/Site/2.3.0/rubygems/commands/install_command.rb:156:in `execute'
/usr/local/Homebrew/Library/Homebrew/utils.rb:230:in `install_gem!'
/usr/local/Homebrew/Library/Homebrew/utils.rb:238:in `install_gem_setup_path!'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/style.rb:21:in `block in install_rubocop'
/usr/local/Homebrew/Library/Homebrew/utils.rb:406:in `capture_stderr'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/style.rb:19:in `install_rubocop'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/style.rb:11:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/abstract_command.rb:35:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:98:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:168:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:132:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'
Error: Kernel.exit
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:179:in `exit'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:179:in `rescue in run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:156:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:132:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'
```

#### Output of `brew cask doctor`

```
==> Homebrew-Cask Version
Homebrew-Cask 1.5.14-3-g94c0d83
caskroom/homebrew-cask (git revision 7bf10; last commit 2018-04-06)
==> macOS
10.13.4
==> SIP
Enabled (custom configuration)

configuration:
apple internal: disabled
kext signing: disabled
filesystem protections: disabled
debugging restrictions: enabled
dtrace restrictions: enabled
nvram protections: enabled
basesystem verification: enabled
==> Java
9.0.4, 1.8.0_162
==> Homebrew-Cask Install Location
<NONE>
==> Homebrew-Cask Staging Location
/usr/local/Caskroom
==> Homebrew-Cask Cached Downloads
~/Library/Caches/Homebrew/Cask (1 files, 75.9MB)
==> Homebrew-Cask Taps:
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-cask (3958 casks)
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-drivers (143 casks)
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-fonts (1159 casks)
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-versions (173 casks)
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-eid (11 casks)
==> Contents of $LOAD_PATH
/usr/local/Homebrew/Library/Homebrew/cask/lib
/usr/local/Homebrew/Library/Homebrew
/Library/Ruby/Gems/2.3.0/gems/did_you_mean-1.0.0/lib
/Library/Ruby/Gems/2.3.0/gems/json-2.1.0/lib
/Library/Ruby/Gems/2.3.0/extensions/universal-darwin-17/2.3.0/json-2.1.0
/Library/Ruby/Gems/2.3.0/gems/psych-3.0.2/lib
/Library/Ruby/Gems/2.3.0/extensions/universal-darwin-17/2.3.0/psych-3.0.2
/Library/Ruby/Site/2.3.0
/Library/Ruby/Site/2.3.0/x86_64-darwin17
/Library/Ruby/Site/2.3.0/universal-darwin17
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0/x86_64-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0/universal-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/x86_64-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/universal-darwin17
==> Environment Variables
LC_ALL="en_US.UTF-8"
PATH="/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/Homebrew/Library/Homebrew/shims/scm"
SHELL="/bin/bash"
```

#### Output of `brew tap`

```
caskroom/cask
caskroom/drivers
caskroom/eid
caskroom/fonts
caskroom/versions
heroku/brew
homebrew/core
homebrew/php
homebrew/services
vitorgalvao/tiny-scripts
yonaskolb/mint
```

#### Output of `ruby -v`
```
12:24 $ ruby -v
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin17]
```

**Comment by claui** [comment]
I’ve had [a similar issue](https://github.com/caskroom/homebrew-cask/issues/44190#issuecomment-381270387) since February. Today I found a fix that works for me.

What happens if you trash the `/usr/local/Homebrew/Library/Homebrew/vendor/bundle` directory (or as a safer alternative, move it to another location)?

I just tried that. Homebrew then went on to recreate the `bundle` directory. Now after being broken for three months, `brew style` finally works again for me.
